app/phone-list/phone-list.component.js

```diff
@@ -7,11 +7,12 @@ angular
       function PhoneListController($http, $scope, uiGridGridMenuService) {
         $scope.phones = [];
         $scope.menuItems = [];
+        $scope.canViewMenu = false;
 
         $scope.gridOptions = {
           data: [],
-          enableGridMenu: true,
-          enableFiltering: true,
+          enableGridMenu: false,
+          enableFiltering: false,
```

Comment:
`enableGridMenu` is `undefined` by default.
`enableFiltering` is `false` by default.

We've explicitly set `enableGridMenu` to false for clarity.

Reply:
Our hope was that by setting `enableFiltering` to `false` that the `Clear All Filters` menu item would not appear when calling `uiGridGridMenuService.getMenuItems($scope.gridApi)`.